src/main/views/courts/tabs/typesContent.njk

```diff
@@ -30,7 +30,7 @@
 
  {% set magistratesCodeError = false %}
   {% if item.checked and item.code | valid === false %}
```

Comment:
When using the | (pipe) are you attempting to pass the `item.code` to a Nunjucks filter? Or instead trying to do an `or` operator within the if statement?

Reply:
passing the item.code to Nunjucks filter